package.json

```diff
@@ -13,6 +13,8 @@
     "react-router-dom": "^5.2.1",
     "react-scripts": "4.0.3",
     "redux": "^4.1.1",
+    "redux-logger": "^3.0.6",
```

Comment:
- [ ] Kindly update the readme following this[ template](https://github.com/microverseinc/readme-template).